wallet/src/ui/app/hooks/index.ts

```diff
@@ -14,3 +14,4 @@ export { default as useOnKeyboardEvent } from './useOnKeyboardEvent';
 export { default as useFileExtentionType } from './useFileExtentionType';
 export { default as useNFTBasicData } from './useNFTBasicData';
 export { useObjectsState } from './useObjectsState';
+export { default as useMutationObserver } from './useMutationObserver';
```

Comment:
I know you're following the existing patterns here but we should try to move away from default exports and just make this a named export.